slices/openjdk-25-jre-headless.yaml

```diff
@@ -0,0 +1,181 @@
+    essential:
+      - openjdk-25-jre-headless_core
+    contents:
+      /usr/lib/jvm/java-25-openjdk-*/lib/classlist:
+
+  # A minimal set of files to run a Java application
+  # excluded dependencies:
+  # - ca-certificates-java_essential - needs chisel support to run
+  #                                    maintainer scripts.
+  # - java-common                    - provides update-alternatives,
+  #                                    not relevant.
+  # - util-linux                     - needed for bash completion
+  #                                    not relevant.
+  # - libjpeg8                       - used in awt, not relevant
+  # - liblcms2-2                     - used in awt, not relevant
+  core:
+    essential:
+      - libc6_libs
+      - libgcc-s1_libs
+      - libstdc++6_libs
+      - zlib1g_libs
+    contents:
+      /etc/java-25-openjdk/jaxp.properties:
+      /etc/java-25-openjdk/jvm-*.cfg:
+      /etc/java-25-openjdk/logging.properties:
+      /etc/java-25-openjdk/net.properties:
+      /etc/java-25-openjdk/security/java.policy:
+      /etc/java-25-openjdk/security/java.security:
+      /usr/lib/jvm/java-25-openjdk-*/bin/java:
+      /usr/lib/jvm/java-25-openjdk-*/conf/jaxp.properties:
+      /usr/lib/jvm/java-25-openjdk-*/conf/logging.properties:
+      /usr/lib/jvm/java-25-openjdk-*/conf/net.properties:
+      /usr/lib/jvm/java-25-openjdk-*/conf/security/java.policy:
+      /usr/lib/jvm/java-25-openjdk-*/conf/security/java.security:
+      /usr/lib/jvm/java-25-openjdk-*/lib/jexec:
+      /usr/lib/jvm/java-25-openjdk-*/lib/jspawnhelper:
+      /usr/lib/jvm/java-25-openjdk-*/lib/jvm.cfg:
+      /usr/lib/jvm/java-25-openjdk-*/lib/libextnet.so:
+      /usr/lib/jvm/java-25-openjdk-*/lib/libjava.so:
+      # lib/modules support
+      /usr/lib/jvm/java-25-openjdk-*/lib/libjimage.so:
+      /usr/lib/jvm/java-25-openjdk-*/lib/libjli.so:
+      /usr/lib/jvm/java-25-openjdk-*/lib/libjsig.so:
+      # required for IO.
+      /usr/lib/jvm/java-25-openjdk-*/lib/libnet.so:
+      # required for IO.
+      /usr/lib/jvm/java-25-openjdk-*/lib/libnio.so:
+      /usr/lib/jvm/java-25-openjdk-*/lib/libverify.so:
+      /usr/lib/jvm/java-25-openjdk-*/lib/libzip.so:
+      /usr/lib/jvm/java-25-openjdk-*/lib/modules:
+      /usr/lib/jvm/java-25-openjdk-*/lib/server/libjsig.so:
+      /usr/lib/jvm/java-25-openjdk-*/lib/server/libjvm.so:
+      /usr/lib/jvm/java-25-openjdk-*/lib/tzdb.dat:
+
+  # Native part of jdk.prefs modules
+  prefs:
+    essential:
+      - openjdk-25-jre-headless_core
+    contents:
+      /usr/lib/jvm/java-25-openjdk-*/lib/libprefs.so:
+
+  # Native part of java.rmi
+  rmi:
+    essential:
+      - openjdk-25-jre-headless_security
+    contents:
+      /usr/lib/jvm/java-25-openjdk-*/bin/rmiregistry:
+      /usr/lib/jvm/java-25-openjdk-*/lib/librmi.so:
+
+  # Native part of the console support
+  console:
+    essential:
+      - openjdk-25-jre-headless_core
+    contents:
+      /usr/lib/jvm/java-25-openjdk-*/lib/lible.so:
```

Comment:
not present anywhere 🤨 

`apt-file search --regexp '/usr/lib/jvm/java-25-openjdk-.*/lib/lible`